src/main/java/seedu/duke/commands/AddCommand.java

```diff
@@ -96,23 +108,29 @@ private void executeAddLink(LinkList links) throws DukeException {
         links.addLink(newLink);
     }
 
+    /**
+     * Adds module to Module List.
+     *
+     * @param modules         Module to add
+     * @throws DukeException If arguments in HashMap are invalid.
+     */
     private void executeAddModule(ModuleList modules) throws DukeException {
         int mc;
         boolean isDone = true;
 
         if (!argumentsMap.containsKey("g") || !argumentsMap.containsKey("mc") || !argumentsMap.containsKey("ay")) {
-            throw new DukeException("OOPS!!! g, mc and ay arguments are required!");
+            throw new DukeException("~Error~ g, mc and ay arguments are required!");
         }
 
         try {
             mc = Integer.parseInt(argumentsMap.get("mc"));
         } catch (NumberFormatException e) {
-            throw new DukeException("OOPS!!! Your MCs are invalid!");
+            throw new DukeException("~Error~ Your MCs are invalid!");
         }
 
         if (argumentsMap.containsKey("d")) {
             if (!argumentsMap.get("d").equals("0") && !argumentsMap.get("d").equals("1")) {
-                throw new DukeException("Your done argument is invalid! Valid values: 1 or 0.");
+                throw new DukeException("~Error~ Your done argument is invalid! Valid values: 1 or 0.");
```

Comment:
I guess maybe can create a variable "EXCEPTION_INVALID_DONE_ARGUMENT" or anything similar in Messages to avoid "magic literals"? But like this should also be fine!